src/header.tsx

```diff
@@ -23,7 +23,7 @@ import { ConnectionDialog } from 'dot-connect/react.js'
 //   MenubarTrigger,
 // } from '@/components/ui/menubar'
 import { useAccounts } from './contexts/AccountsContext'
-import { useEffect, useState } from 'react'
+import React, { useEffect, useState } from 'react'
```

Comment:
lets keep it simple:
```suggestion
import { Fragment, useEffect, useState } from 'react'
```